python/cugraph-service/server/cugraph_service_server/testing/utils.py

```diff
@@ -34,6 +34,13 @@ def create_tmp_extension_dir(file_contents, file_name="my_extension.py"):
     return tmp_extension_dir
 
 
+_failed_subproc_err_msg = (
```

Comment:
These are coming from https://github.com/rapidsai/cugraph/pull/3012 right ? 

We should let that merge in first .

Reply:
right, this is dependent on 3012; I'll make a note of that.